lib/src/style/Theme.cpp

```diff
@@ -435,8 +435,23 @@ QColor const& Theme::comboBoxForegroundColor(MouseState const mouse) const {
   return buttonForegroundColor(mouse, ColorRole::Neutral);
 }
 
+QColor const& Theme::comboBoxTextColor(MouseState const mouse, Status const status) const {
```

Comment:
As talked with @christophe-thiery-fw , all these methods can go to `Qlementine`  and stay `virtual`, to allow subclasses to customize the behavior.